NWNXLib/API/API/CNWSScriptVar.hpp

```diff
@@ -23,7 +23,6 @@ struct CNWSScriptVar
     CScriptLocation m_location;
 
     CNWSScriptVar();
-    CNWSScriptVar(const CNWSScriptVar &);
```

Comment:
Ugh, this whole class is missing from FunctionsLinux.hpp. The API _is_ correct. I'll add the offsets.